draft-ietf-suit-information-model.md

```diff
@@ -625,7 +655,7 @@ Implemented By: [Vendor ID Condition](#element-vendor-id), [Class ID Condition](
 
 ### REQ.SEC.RIGHTS: Rights Require Authenticity {#req-sec-rights}
 
-If a device grants different rights to different actors, exercising those rights MUST be accompanied by proof of those rights, in the form of proof of authenticity. Authenticity mechanisms such as those required in [REQ.SEC.AUTHENTIC](#req-sec-authentic) are acceptable but need to follow the end-to-end security model.
+If a device grants different rights to different actors, exercising those rights MUST be accompanied by proof of those rights, in the form of proof of authenticity. Authenticity mechanisms such as those required in [REQ.SEC.AUTHENTIC](#req-sec-authentic) can be used to prove authenticity.
 
 For example, if a device has a policy that requires that firmware have both an Authorship right and a Qualification right and if that device grants Authorship and Qualification rights to different parties, such as a Device Operator and a Network Operator, respectively, then the firmware cannot be installed without proof of rights from both the Device and the Network Operator.
```

Comment:
I still think the end should say "from both the Device Operator and the Network Operator", i.e., insert "Operator" before "and".